Qubit Server/dev/src/main/java/org/labrad/qubits/FpgaModelDac.java

```diff
@@ -1,27 +1,28 @@
 package org.labrad.qubits;
 
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.Future;
-
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.labrad.data.Data;
+import org.labrad.data.Request;
 import org.labrad.qubits.channels.TriggerChannel;
+import org.labrad.qubits.controller.FpgaController;
+import org.labrad.qubits.controller.JumpTableController;
+import org.labrad.qubits.controller.MemoryController;
 import org.labrad.qubits.enums.DacTriggerId;
-import org.labrad.qubits.mem.CallSramCommand;
-import org.labrad.qubits.mem.CallSramDualBlockCommand;
-import org.labrad.qubits.mem.DelayCommand;
-import org.labrad.qubits.mem.EndSequenceCommand;
-import org.labrad.qubits.mem.MemoryCommand;
-import org.labrad.qubits.mem.NoopCommand;
-import org.labrad.qubits.mem.StartTimerCommand;
-import org.labrad.qubits.mem.StopTimerCommand;
 import org.labrad.qubits.proxies.DeconvolutionProxy;
 import org.labrad.qubits.resources.DacBoard;
 
-import com.google.common.base.Preconditions;
-import com.google.common.collect.Lists;
-import com.google.common.collect.Maps;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Future;
```

Comment:
Is this intellij just reordering imports? We should agree on an ordering and set up all the IDEs to do it the same way to avoid superfluous diffs; my preference is just all the imports in one block, sorted alphabetically.